src/@utils/ocean.ts

```diff
@@ -72,3 +74,23 @@ export async function getOceanBalance(
     LoggerInstance.error(`ERROR: Failed to get the balance: ${e.message}`)
   }
 }
+
+export async function convertGasFeesToOcean(
```

Comment:
not sure we have to do all this conversion as making our `usePrices()` hook also deliver ETH prices is super trivial as the Coingecko API allows passing multiple token IDs, so would stay just one request. But guess we would have to refactor some places then

Reply:
Ok, I will look into it. At this level also, we'll need to make some adjustments based on changes to [`Prices.tsx`](https://github.com/oceanprotocol/market/pull/1132/files). We'll need to add the fix [here](https://github.com/oceanprotocol/market/blob/feature/issue-1066-gas-estimation-buy/src/components/Asset/AssetActions/Download.tsx#L72:L72) to make sure these calculations are done for matic too.

Reply:
yup, trivial, with this in `@context/Prices`:

```js
const tokenIds = 'ocean-protocol,ethereum,matic-network'
```

we get this back:

<img width="794" alt="Screen Shot 2022-04-15 at 17 03 52" src="https://user-images.githubusercontent.com/90316/163593491-84f3658f-498b-4b5f-a2c0-c5f11301949c.png">

Requires changing a bit data structure in `usePrices()` as it looks like though